docs/samples/browser-plugin-meetings/app.js

```diff
@@ -263,8 +263,8 @@ const displayMeetingStatusElm = document.querySelector('#display-meeting-status'
 const spaceIDError = `Using the space ID as a destination is no longer supported. Please refer to the <a href="https://github.com/webex/webex-js-sdk/wiki/Migration-to-Unified-Space-Meetings" target="_blank">migration guide</a> to migrate to use the meeting ID or SIP address.`;
 const BNR = 'BNR';
 const VBG = 'VBG';
-const blurVBGImageUrl = 'https://webex.github.io/webex-js-sdk/api/assets/vbg_image.jpg';
-const blurVBGVideoUrl = 'https://webex.github.io/webex-js-sdk/api/assets/clouds.5b57454a.mp4';
+const blurVBGImageUrl = '/api/assets/vbg_image.jpg';
```

Comment:
is the crypto thing for this change? how do we properly access the url now? maybe a comment here would help? is it because the effect lib now does this internally?

Reply:
No, the crypto thing is for the unit tests (sorry, I should have clarified this in my comment above, I'll edit it). The URL change is because the original URLs don't actually exist, so I've changed them to use relative URLs instead. Let me know if this is a problem; they seem to work when running the samples locally.